### DISCUSSION TITLE
how to pinned button push up and pull down for tabs

### DISCUSSION THREAD

**Discussion by candrapersada** [floccusaddon/floccus#1654] [General]
how to pinned button push up and pull down so as not to lose the button in menu?
![image](https://github.com/floccusaddon/floccus/assets/36833664/7dcb0292-43d1-46a9-aa7c-e1d253005ee6)

**Comment by marcelklehr** [comment]
@candrapersada You can write in your own language but please be more specific than this :)

**Comment by candrapersada** [comment]
how do I pin the tabs sync button so I don't lose the tabs sync button in the shortcut menu?

**Reply by marcelklehr** [reply]
Where do you want to pin the tab sync button to? Is it that you want to always pull down? Perhaps a larger screenshot would help?

**Reply by candrapersada** [reply]
![image](https://github.com/floccusaddon/floccus/assets/36833664/b3bef6a1-53e8-42e2-84a6-81c11fe6de43)

**Reply by marcelklehr** [reply]
So, the issue is that the popup is too small? Which browser are you on?

**Reply by marcelklehr** [reply]
Or is the issue that you disabled auto sync and that causes the tab profile to be hidden beyond the fold and you have to scroll to be able to sync?

**Reply by candrapersada** [reply]
> Or is the issue that you disabled auto sync and that causes the tab profile to be hidden beyond the fold and you have to scroll to be able to sync?

Yes because the tabs problem is still not resolved, have to use it manually. sync tabs https://github.com/floccusaddon/floccus/issues/850#issuecomment-1065752318
